rad/src/operators/string.rs

```diff
@@ -148,25 +149,40 @@ pub fn radon_trim(input: &RadonString) -> String {
     }
 }
 
-pub fn to_float(input: &RadonString) -> Result<RadonFloat, RadError> {
+pub fn to_bool(input: &RadonString) -> Result<RadonBoolean, RadError> {
     let str_value = radon_trim(input);
-    f64::from_str(&str_value)
+    bool::from_str(&str_value)
+        .map(RadonBoolean::from)
+        .map_err(Into::into)
+}
+
+/// Converts a `RadonString` into a `RadonFloat`, provided that the input string actually represents
+/// a valid floating point number.
+pub fn as_float(input: &RadonString, args: &Option<Vec<Value>>) -> Result<RadonFloat, RadError> {
+    f64::from_str(&as_numeric_string(input, &args.clone().unwrap_or_default()))
```

Comment:
You can use `Option::as_deref` to convert `&Option<Vec<T>>` into `&Option<&[T]>`:

```suggestion
    f64::from_str(&as_numeric_string(input, args.as_deref().unwrap_or_default()))
```

Same below in as_integer.